yok.ts

```diff
@@ -192,6 +192,7 @@ export class Yok implements IInjector {
 	private createHierarchicalCommand(name: string) {
 		let factory = () => {
 			return {
+				disableAnalytics: true,
```

Comment:
The `createHierarchicalCommand` is a little magic for commands defined as `<part1>|<part2>` - during parsing of the command line, CLI tries to check each argument for a valid command and if there's no such command, it will fail.
So in case you write `<cli name> part1 part2` CLI gets the part1 and tries to resolve a command called part1. You have not defined such in the code. This is resolved with the current code. It's been mistakenly forgotten that such commands should not be tracked.

As per the relation with proxy commands - again - when you execute `<cli name> proxy set ...`, CLI will first resolve this command (during evaluating the proxy part of the command line. As it does not have disabledAnalytics, CLI will try to track that this command is used and the request will fail (as the proxy is still not set).